src/server/controllers/ProjectService.ts

```diff
@@ -0,0 +1,112 @@
+import { IProjectModel, Projects } from '../models/ProjectModel';
+import { ProjectCategories } from '../models/ProjectCategory';
+import * as Sequelize from 'sequelize';
+import { Logger } from '../util/Logger';
+import * as Promise from 'bluebird';
+import { printAndThrowError } from '../util/common';
+
+const logger = Logger('controllers/ProjectService');
+
+export class ProjectService {
+
+  public static find(getCategories?: boolean): Promise<IProjectModel[]> {
```

Comment:
Did you verify if the getCategories argument works? I don't see you ever passing a non-null value into this anywhere..